karoo_gp/test/util.py

```diff
@@ -0,0 +1,75 @@
+import hashlib, os, sys
+import numpy as np
+import sklearn.model_selection as skcv
+
+def hasher(msg):
+    if not isinstance(msg, bytes):
+        msg = str(msg).encode('utf-8')
+    hasher = hashlib.md5()
+    hasher.update(msg)
+    return hasher.hexdigest()
```

Comment:
Eventually this would probably be better as `Tree.__hash__` or `Population.__hash__` methods.  For other situations it might be better to use other methods, since the hash is not very useful when the tests fail.